packages/react/carbon.yml

```diff
@@ -0,0 +1,699 @@
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-structuredlist
+    tabs:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/tabs/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-tabs
+    tag:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/tag/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-tag
+    text-area:
+      status: stable
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-textarea
+    text-input:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/text-input/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-textinput
+    theme:
+      status: stable
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-theme--default
+    tile:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/tile/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-tile
+    time-picker:
+      status: stable
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-timepicker
+    toggle:
+      status: stable
+      externalDocsUrl: https://www.carbondesignsystem.com/components/toggle/usage
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-toggle
+    toggletip:
+      name: Toggletip
+      framework: react
+      externalDocsUrl: https://www.carbondesignsystem.com/components/toggletip/usage/
+      status: stable
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-toggletip
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-element 
```

Comment:
Delete content-element. Add system-feedback and input-control.

Reply:
@kingtraceyj would you be able to look at this one and give your thoughts, too?

Reply:
@aubrey-oneal toggletip and tooltip are very closely related. the main difference is that a toggletip can contain buttons and links and other interactive elements where a tooltip should not. i think we should use the same tag we used for tooltip.